sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/DoFnAdapters.java

```diff
@@ -86,6 +86,7 @@ public void setup() throws Exception {
 
     @Override
     public void startBundle(Context c) throws Exception {
+      fn.prepareForProcessing();
```

Comment:
The current PR is not moving the call - it was moved in the previous PR refactoring DoFnReflector. I just accidentally removed the call in one of the intermediate commits (which caused the test failures) and restored. I think we discussed offline that when DoFnAdapters goes away, we'll just invoke .prepareForProcessing() explicitly in the new codepath (whatever it may be)? I don't see the benefit in invoking it via generated code rather than explicitly.